src/main/java/de/rwth/idsg/steve/repository/impl/WebUserRepositoryImpl.java

```diff
@@ -40,6 +48,7 @@
 public class WebUserRepositoryImpl implements WebUserRepository {
 
     private final DSLContext ctx;
+    private final ObjectMapper jacksonObjectMapper;
```

Comment:
please do not leak ObjectMapper into repository layer.

my first impl was like this (as there was no WebUserService). but during the refactor, i moved all business decisions and non-database actions into service layer.